packages/dotnet-sdk/src/TeamsFx/Conversation/NotificationBot.cs

```diff
@@ -75,8 +76,7 @@ await _adapter.ContinueConversationAsync
                 (
                     _botAppId,
                     reference,
-                    async (context, ct) =>
-                    {
+                    async (context, ct) => {
```

Comment:
nit: keep the open curly in a new line for csharp?

Reply:
It's formatted by Visual Studio.